TRANSFORMERS/VECTOR_MANIPULATION/VECTOR_INDEXING/VECTOR_INDEXING_test_.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+from flojoy import Vector, Scalar
+
+
+def test_VECTOR_INDEXING(mock_flojoy_decorator):
+    import VECTOR_INDEXING
+
+    x = np.ones(5)
+    x[0] = 5
+
+    element = Vector(v=x)
+    res = VECTOR_INDEXING.VECTOR_INDEXING(element)
+
+    assert np.array_equal(res.c, 5)
```

Comment:
Could you also add a test for cases when index out of bound and assert the error msg

Reply:
I'm not sure how to do this. I can't find an example.

Reply:
I believe you can use pytests' `raises` context manager